.golangci.example.yml

```diff
@@ -24,7 +24,7 @@ linters:
     - goimports
     - gosec
     - gosimple
-    - golint
+    - revive
```

Comment:
there is no revive configured...

Reply:
what do you mean? `revive` is part of `golangici-lint`?